ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.13-nightly-arm.yaml

```diff
@@ -1,15 +1,16 @@
+base_images:
```

Comment:
We might need to verify one of these jobs as well, to see if the image is architecture specific.